### PR TITLE
:zero::seven::zero: use latest frontend library version

### PR DESCRIPTION
**Pull request by ajthomascouk**

### CODE REVIEW

File: scss-0.7.0/environment/tools/functions/_asset-urls.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (st3v3nhunt) [inline scss-0.7.0/environment/tools/functions/_asset-urls.scss]: Unnecessary blank line, please remove.

Comment (ajthomascouk) [reply scss-0.7.0/environment/tools/functions/_asset-urls.scss]: This is in the FEL. Whats your thoughts on me adding this to the FEL and we catch it next time?

Comment (st3v3nhunt) [reply scss-0.7.0/environment/tools/functions/_asset-urls.scss]: Ah ok. Sure, if you get it sorted.

**Comment by st3v3nhunt** [comment]
The branch needs rebasing now please.

File: scss-0.7.0/environment/tools/functions/_asset-urls.scss (hunk 1 above)
Comment (ajthomascouk) [reply scss-0.7.0/environment/tools/functions/_asset-urls.scss]: You know what.... lets just get it sorted now...

**Pull request merged**